src/test/java/org/apache/maven/plugins/dependency/resolvers/ExcludeReactorProjectsDependencyFilterTest.java

```diff
@@ -124,7 +119,7 @@ public Dependency getDependency() {
 
         ArgumentCaptor<String> captor = ArgumentCaptor.forClass( String.class );
         verify( log ).debug( captor.capture() );
-        assertThat( captor.getValue(), containsString( "Skipped dependency" ) );
+        assertTrue( captor.getValue().contains( "Skipped dependency" ) );
```

Comment:
revert This.
`assertTrue` and the old JUnit assertios are the bad practice.

Reply:
Reference?